docs/user/rstudio/ide/guide/pkg-devel/writing-packages.qmd

```diff
@@ -56,27 +60,29 @@ If you don't already have these tools installed on your system please consult th
 
 ### Creating a New Package
 
-To create a new package use the **Create Project** command (available on the Projects menu and on the global toolbar) and select the **New Directory** option. Then on the following screen specify the project type as **R Package**:
+To create a new package use the **Create Project** command (available on the **Projects** menu and on the global toolbar) and select the **New Directory** option. Then on the following screen specify the project type as **R Package**:
 
-![](images/create-r-pkg.png){alt=""}
+![](images/create-r-pkg.png){fig-alt="A screenshot of the New Project Wizard"}
 
-Note that if you have existing R scripts that you'd like to use as the basis for the new package you can specify them here and they'll be included in the new package.
+::: callout-tip
+If you have existing R scripts that you'd like to use as the basis for the new package you can specify them here and they'll be included in the new package.
+:::
 
 ### Working with an Existing Package
 
-To enable RStudio's package development tools for an existing package you should do the following:
+To enable RStudio's package development tools for an existing package do the following:
 
-1.  Create a new [RStudio Project](https://support.rstudio.com/hc/en-us/articles/200526207-Using-Projects) associated with the package's directory.
+1.  Create a new [RStudio Project](../code/projects.html) associated with the package's directory.
 
-2.  If the package DESCRIPTION file is located either in the project's root directory or at pkg/DESCRIPTION then it will be automatically discovered.
+2.  If the package `DESCRIPTION` file is located either in the project's root directory or at `pkg/DESCRIPTION` then it will be automatically discovered.
 
-3.  Alternatively, go to **Project Options : Build Tools**, select "Package" as the project build tools type, and then specify the the subdirectory containing the package's DESCRIPTION file.
+3.  Alternatively, go to **Tools** \> **Project Options** \> **Build Tools**, select "Package" as the project build tools type, and then specify the the subdirectory containing the package's `DESCRIPTION` file.
 
 ## Building a Package
 
 To work with packages in RStudio you use the **Build** pane, which includes a variety of tools for building and testing packages. While iteratively developing a package in RStudio, you typically use the **Build and Reload** command to re-build the package and reload it in a fresh R session:
```

Comment:
I think this is no longer Build and Reload for packages. Seems to now be called `Clean and Install` in recent releases

Reply:
Resolved, thanks!